### DISCUSSION TITLE
156

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#96] [General]
# 潮流周刊第156期 - 富士山下

封面图来源于从大阪到东京新干线途中所拍，恰逢天气很好，拍到了远处的富士山 🗻，很很很不错，大概 5min 中观看时间。

https://weekly.tw93.fun/posts/156-%E5%AF%8C%E5%A3%AB%E5%B1%B1%E4%B8%8B

**Comment by zsj1029** [comment]
什么时候支持暗色模式，太白刺眼，习惯暗色了

**Reply by tw93** [reply]
推荐试试这个 https://darkreader.org/ 效果很不错

**Comment by Wuli-Blizzard** [comment]
富士山好大！

**Reply by tw93** [reply]
哈哈哈 只是手机拉的远

**Comment by Wuli-Blizzard** [comment]
😊哈哈哈多拍点爱看(=^▽^=)🥰

**Comment by Marilyn2022** [comment]
测试一下
